galsim/wfirst/wfirst_detectors.py

```diff
@@ -149,20 +173,19 @@ def allDetectorEffects(img, prev_exposures=(), rng=None, exptime=default_exptime
     dark_noise = galsim.DeviateNoise(galsim.PoissonDeviate(rng, dark_current))
     img.addNoise(dark_noise)
 
-    # Persistence (use WFIRST coefficients)
-    prev_exposures = list(prev_exposures)
-    applyPersistence(img,prev_exposures)
-
-    # Update the 'prev_exposures' queue
-    ncoeff = len(galsim.wfirst.persistence_coefficients)
-    prev_exposures = [img.copy()] + prev_exposures[:ncoeff-1]
-
     # Nonlinearity (use WFIRST routine).
     applyNonlinearity(img)
 
     # IPC (use WFIRST routine).
     applyIPC(img)
 
+    # Persistence (use WFIRST H4RG-lo fermi model)
+    prev_exposures = list(prev_exposures)
+    applyPersistence(img, prev_exposures, method='fermi')
```

Comment:
I thought persistence worked like an extra dark current.  Why did this move?  Shouldn't it be applied prior to NL and IPC?